pgmpy/estimators/ExpertKnowledge.py

```diff
@@ -0,0 +1,41 @@
+class ExpertKnowledge:
+    """
+    Class to specify expert knowledge for causal discovery algorithms.

```

Comment:
We follow the numpydoc style for docstrings: https://numpydoc.readthedocs.io/en/latest/format.html. The standard format for the description should be a one sentence summary, then blank line, and a paragraph of details.